packages/components/src/vue/components/notification-area.vue

```diff
@@ -41,6 +41,32 @@
           <close-icon/>
       </div>
     </div>
+    <div
```

Comment:
Kirill, what do you think about splitting notification area content to several components.
For now it has such a specific structure, that we should create a specific separate area for each notification:
hint, notification, auth notification and now cedict notification.
This component becomes bigger and bigger, it has duplicted code and it is not easy to ready.
May be it is a good time to optimize it?
Or I could do it while would work on notification component for toolbar.
@kirlat , @balmas , what do you think?

Reply:
I think it would be great to split notifications into several components, so each single notification would be represented by a single instance of that component. With more and more parts of an application generating notifications we should have an architecture where each part (such as CEDICT) would be able to create a notification easily.

I think it's also good for notifications to be standardized using the same overall design. However, notifications may differ in colors. Each notification might have several elements that could be turned on or off for each particular notification instance. Those elements might include a text message, a progress bar, one or several action buttons, a close icon, and whatever else we might find worth including.

What do you think?

Reply:
yes I agree with these suggestions. Irina if you want to start working on the notification component refactoring this is fine with me.